plugins/elasticsearch/elasticsearch_test.go

```diff
@@ -0,0 +1,258 @@
+package elasticsearch
+
+import (
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/influxdb/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+const indicesResponse = `
+{
+  "cluster_name": "es-testcluster",
+  "nodes": {
+    "SDFsfSDFsdfFSDSDfSFDSDF": {
+      "timestamp": 1436365550135,
```

Comment:
I prefer trusting the timestamp from Telegraf. What if one of the nodes has a time that is slightly off, different timezone (by accident)...?